test/internal/info_plists/BUILD.bazel

```diff
@@ -0,0 +1,6 @@
+load(":get_file_from_objc_provider_tests.bzl", "get_file_from_objc_provider_test_suite")
+load(":get_file_tests.bzl", "get_file_test_suite")
+
+get_file_from_objc_provider_test_suite()
```

Comment:
Can we pass the name of the suites here like in the other tests?

Reply:
Sure. Curious. What is the benefit of doing that way?

Reply:
Mainly following this: https://github.com/bazelbuild/rules_apple/commit/e3ebb8f4b7b8ed8b155b49d434bc7e13e41ba092

Reply:
Interesting. I am surprised that Buildifier did not yell at me. Thanks.